core/modules/views/plugins/views_plugin_pager_mini.inc

```diff
@@ -26,7 +26,7 @@ class views_plugin_pager_mini extends views_plugin_pager_full {
   function option_definition() {
     $options = parent::option_definition();
 
-    unset($options['quantity']);
+    $options['quantity'] = NULL;
     unset($options['tags']['first']);
     unset($options['tags']['last']);
```

Comment:
What's wrong with just leaving the parent form's default values in place? This'll prevent this issue of missing values and possible future relayed issues with the other unneeded values.

```suggestion
```

Reply:
> What's wrong with just leaving the parent form's default values in place?

Then the form items for first/last/quantity would show up in the settings form - they shouldn't, as the mini pager doesn't use these settings. It would be OK to keep the "unset($form['quantity']);" because that simply removes the form item, which shouldn't cause any problems. Setting it to NULL doesn't do any harm, either.

The `$options['quantity'] = NULL;` in function option_definition() is the essential change to fix the notice.

Reply:
I'll leave things as they are for now 😅